layouts/partials/footer.html

```diff
@@ -7,7 +8,7 @@
              <a class="usa-footer-primary-link" href="/docs/help/">cloud.gov help</a>
            </li>
            <li class="usa-width-one-fourth usa-footer-primary-content">
-             <a class="usa-footer-primary-link" href="https://github.com/18F/cg-site">Edit this page</a>
+             <a class="usa-footer-primary-link" href="https://github.com/18F/cg-site">Source code <br>(public domain)</a>
```

Comment:
I think @brittag 's intent was to make this a link to go edit this particular page, as used to be possible via the "Refine this page" item in the menu; is that right? 

That's menu item is gone now, and simply linking to the top of the repository here means that people have to understand Hugo and go navigate through the repository to find the right source file to even to suggest a typo fix. That adds enough friction that we're likely to get fewer contributors! (At least, it's enough friction that it dissuades me, and I imagine others feel the same.)